JASP-Desktop/widgets/listmodel.cpp

```diff
@@ -465,6 +479,37 @@ const QString &ListModel::name() const
 	return _listView->name();
 }
 
+const Terms &ListModel::terms(const QString &what) const
+{
+	if (what.startsWith("type="))
+	{
+		static Terms terms;
+
+		QStringList typesStr = what.right(what.length() - 5).toLower().split("|");
+		QList<columnType> types;
+
+		for (const QString& typeStr : typesStr)
+		{
+			columnType type = columnType::unknown;
+			try { type = columnTypeFromQString(typeStr); } catch (...) {}
```

Comment:
Do you think it would be useful to create a second version of `enumFromQString` that gives back a default enumval if the one the user gives is crap?
Would make this code prettier!
And it wouldn't be that hard either.

Reply:
Good idee!

Reply:
Will you try it? Then I won't be the only one who has a clue about `enumutilities.h` ;)

Reply:
You already did it! Haha